azkaban-common/src/main/java/azkaban/AzkabanCommonModule.java

```diff
@@ -158,4 +163,10 @@ public FileSystem createHadoopFileSystem(final Configuration hadoopConf) {
   public QueryRunner createQueryRunner(final AzkabanDataSource dataSource) {
     return new QueryRunner(dataSource);
   }
+
+  @Provides
```

Comment:
I was trying to google the difference between using `@provide` and `to bind`, but failed. Some online resources say that there'w no difference between the two approaches. I believe @suvodeep-pyne has better understanding on this case.